src/components/Main/SingleArticle/articlesComments.jsx

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Loading from "../../UI/Loading";
+import Error from "../../UI/error";
+import './singelArticle.css'; // Import additional CSS if needed
+
+function ArticleComments({ article_id }) {
+    const [comments, setComments] = useState([]);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        axios.get(`https://nc-news-vvdv.onrender.com/api/articles/${article_id}/comments`)
+            .then(({ data }) => setComments(data.comments))
+            .catch((err) => setError(err));
+    }, [article_id]);
+
+
+    if (error) return <Error error={error} />;
+    if (comments.length === 0) return <Loading />;
```

Comment:
What happens if an article doesn't have any comments. It would be better to handle loading manually.